### PR TITLE
Convert nupic.bindings egg to wheel for deployment

### LINKED ISSUE #2413: Master build deployment failure, missing "nupic.bindings"
See: https://travis-ci.org/numenta/nupic/jobs/75138342#L6712

```
Downloading/unpacking nupic.bindings==0.1.0 (from nupic==0.2.7.dev0)
  Could not find any downloads that satisfy the requirement nupic.bindings==0.1.0 (from nupic==0.2.7.dev0)
Cleaning up...
No distributions at all found for nupic.bindings==0.1.0 (from nupic==0.2.7.dev0)
Storing complete log in /home/travis/.pip/pip.log
The command ". ./ci/travis/before_deploy.sh" failed and exited with 1 during .
```

CC @akhilaananthram @scottpurdy

### PR DESCRIPTION
**Pull request by akhilaananthram**
To deploy nupic, we need the wheel for nupic.bindings and the path to it.  This converts it to a wheel and gets the path.

Fixes #2413

@rhyolight @scottpurdy

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -212,7 +213,12 @@ def findRequirements(nupicCoreReleaseDir):
     dependencies.append(wheel)
   eggFiles = glob.glob(os.path.join(nupicCoreReleaseDir, "*.egg"))
   for egg in eggFiles:
-    dependencies.append(egg)
+    filename = os.path.splitext(egg)[0]
+    currentDir = os.getcwd()
+    os.chdir(nupicCoreReleaseDir)
```
Comment (scottpurdy) [inline setup.py]: Can you use `--dest-dir` instead of changing into the directory?

**Comment by scottpurdy** [comment]
done with review

**Comment by breznak** [comment]
@rhyolight this is weird https://travis-ci.org/numenta/nupic/jobs/75174495#L594

**Comment by breznak** [comment]
Btw, I've tested this and getting the same error as in #2413

**Pull request closed without merging**